services/thanos/15.8.0/defaults/cm.yaml

```
@@ -9,7 +9,7 @@ data:
     image:
       registry: docker.io
       repository: bitnami/thanos
-      tag: 0.38.0-debian-12-r0
+      tag: 0.36.1-debian-12-r3
```

```suggestion
      tag: 0.38.0-debian-12-r0
```

Do we need this change?

---

Yes based on docummentation https://artifacthub.io/packages/helm/bitnami/external-dns/8.5.0
OCI helm repo uses this image docker.io/bitnami/thanos:0.36.1-debian-12-r3

---

It was set to the higher image version for a reason probably